tests/functional/firefox/test_enterprise.py

```diff
@@ -11,3 +11,11 @@
 def test_primary_download_button_displayed(base_url, selenium):
     page = EnterprisePage(selenium, base_url).open()
     assert page.is_primary_download_button_displayed
+
+
+@pytest.mark.nondestructive
+def test_download_lists_displayed(base_url, selenium):
+    page = EnterprisePage(selenium, base_url).open()
+    assert page.is_win64_download_list_displayed
```

Comment:
Rather than displaying that each of these menus is displayed, I think we can go a little further and click each one to assert that the menus open. Take a look at this test as an example:

https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/test_home.py#L13-L14